packages/veraswap-app/src/components/TransactionStatusModal.tsx

```diff
@@ -46,7 +48,12 @@ export function TransactionStatusModal({
                     // TODO: fix messageId return `https://sid.testnet.routescan.io/crosstransactions/${hashes.bridge}`;
                 }
                 */
-                return `https://explorer.hyperlane.xyz/message/${hashes.bridge}`;
+                return isNativeBridge
+                    ? networkType === "testnet"
+                        ? `https://test.orbiter.finance/en?src_chain=${chains?.source?.id}&tgt_chain=${chains?.destination?.id}&src_token=ETH`
+                        : `https://www.orbiter.finance/history?src_chain=${chains?.source?.id}&tgt_chain=${chains?.destination?.id}&src_token=ETH`
+                    : `https://explorer.hyperlane.xyz/message/${hashes.bridge}`;
```

Comment:
Tried to message to see if we can get proper query params. In the meantime maybe less confusing to put the explorer url + tx hash of the remote chain

Reply:
Why are we redirecting to orbiter? The link doesn't show the history anyways and this just seems confusing for the user, especially since Orbiter also has a swap UI.

Reply:
Agree with @OscBacon just put the block explorer link.